src/components/ui/map/legend/component.js

```diff
@@ -10,6 +10,7 @@ import {
 
 // styles
 import './styles.scss';
+import { RETURN_PERIOD_OPTIONS } from 'components/hazard/map/legend/constants';
```

Comment:
Why do we need to import these now when we didn't before?

Reply:
Beats me.